Sources/tart/Commands/VNC.swift

```diff
@@ -0,0 +1,64 @@
+import ArgumentParser
+import Foundation
+import Network
+import SystemConfiguration
+
+struct VNCCredentialsNotFound: Error {
+}
+
+struct VNC: AsyncParsableCommand {
```

Comment:
> This command will only work if a VM was started via `run --vnc` and we saved the credentials.

This PR changes things a little and makes the VNC server always available because it's cheap now.

Reply:
So, should it be completely disabled by default then? And only enabled with something like `--vnc-plus`? While the `--vnc` would use the mechanism you've mentioned above:

> Note that you can still VNC into a VM if _Remote Login_ option is enabled inside the VM.

Reply:
Initially I though of just replacing the existing logic with the private API in the `if vnc` statement.

Reply:
Implemented in https://github.com/cirruslabs/tart/pull/126/commits/f5a491111bcf05fceb604f2f3e1c1e076cd6e381 and  https://github.com/cirruslabs/tart/pull/126/commits/ff27894b0afff93f032c3e6270f125af79d3ae3f.